saviSimulation/savi/jason_processing/WifiAntenna.java

```diff
@@ -85,10 +85,8 @@ public static void setPerceptionDistance(double value) {
 		wifiPerceptionDistance=value;
 	}
 	
-	public static void setSeed(int seed) {
-		if(seed != -1) {
-			rand = new Random(seed);
-		}
+	public static void setSeed(Random ran) {
+		rand = ran;
 	}
```

Comment:
If WifiAntenna were a WorldObject, this method could be inherited. Is there anything preventing WifiAntenna from being a WorldObject? (I'm not sure if we've discussed this before).